akka-http/src/main/scala/akka/http/javadsl/server/Directives.scala

```diff
@@ -23,6 +23,7 @@ object Directives extends AllDirectives {
   // These are repeated here since sometimes (?) the Scala compiler won't actually generate java-compatible
   // signatures for varargs methods, making them show up as Seq<Object> instead of T... in Java.
 
+  @Deprecated
   @varargs override def route(alternatives: Route*): Route =
```

Comment:
javadoc is inherited from RouteDirectives right? okey then (to make sure the deprecation reason shows up

Reply:
Yes, javadocs will be taken from a superclass.